### DISCUSSION TITLE
npm i 安装失败 fatal - Error: Cannot find module '../pages/access' from '/root/workspace/src/pages'

### DISCUSSION THREAD

**Discussion by Samsara-2017** [umijs/umi#11708] [General]
node v16.18.1

<img width="1153" alt="image" src="https://github.com/umijs/umi/assets/29455938/bb0a82a0-4364-454b-b302-bc92c22fa302">

**Comment by fz6m** [comment]
配置文件路由表路径写错了，找不到文件，检查一下。
